lg_earth/src/lg_earth/kmlsync.py

```diff
@@ -330,10 +330,24 @@ class KmlQueryHandler(tornado.web.RequestHandler):
     def initialize(self):
         self.playtour = self.application.playtour
         self.playtour_service = self.application.playtour_service
+        self.planet = self.application.planet
+        self.planet_service = self.application.planet_service
+        self.get_planet = self.application.get_planet
+
+    def _wait_for_planet(self, planet):
```

Comment:
Oh I thought that we're observing the "desired" state instead of "actual" state. Thas OK then.

Another possible problem - we're waiting in the loop for 5 seconds but is this wait blocking operation?

If it's blocking (which means that **whole** ros node is waiting) then bad but acceptable in this case ( GE <3 ).

If it's non-blocking then kmlsync will accept and process incoming messages which may lead to random behavior.

Can you please add some tests here?